src/components/reviews/reviewPhoto.jsx

```diff
@@ -14,7 +15,8 @@ class ReviewPhoto extends React.Component {
     this.exitModalBound = this.exitModal.bind(this);
   }
 
-  openModal() {
+  openModal(event) {
```

Comment:
I didn't think of directly using the event as part of the interaction recording. Good idea.